nixos/roles/ceph/osd.nix

```diff
@@ -223,6 +225,8 @@ in
       } // osdServiceDeps;
 
       systemd.services."fc-ceph-osd@" = rec {
+        enable = ! config.flyingcircus.services.ceph.server.passive;
```

Comment:
I'm not fully certain what `enable` causes for services that we only want to pull in by other depending units. Need to check that.

Reply:
This is fine, the default is `true` anyways, setting it to `false` just equals a mask.